init.sh

```diff
@@ -7,10 +7,16 @@ set -e
 #
 
 MODE=${MODE:-BACKUP}
+TARBALL=${TARBALL:-}
  
 case "${MODE^^}" in
     'BACKUP')
-        OPTIONS=${OPTIONS:--c}
+        if [[ "${MODE^^}" == "BACKUP" ]] && [[ "${TARBALL^^}" != "" ]]
```

Comment:
You can remove the condition `[[ "${MODE^^}" == "BACKUP" ]]` as the `MODE` is already checked in the switch/case statement.